src/utils/requests/BillRequests.js

```diff
@@ -0,0 +1,19 @@
+const {URL} = require("../../config");
+/**
+ * @function getBill
+ * @description Function that calls the /bills route to return the bills associated to the user. With bearer token.
+ * @param {String} token
+ */
+export const getBill = async (token) => {
```

Comment:
You're missing some tests. 

refer to: https://github.com/dannylekim/billsnapFront/pull/17/files#diff-483fbe43c486ff2cae5828bd7b7fb714